### PR TITLE
Add WIT code to repo

### PR DESCRIPTION
**Pull request by jameswex**
This PR adds all WIT code to repo, from the the TB repo.

Tested:
- Can still build witwidget pip/npm packages
- Demos build/run correctly
- UTs all pass
- Can build new wit_tensorboard pip package for TB dynamic plugin (not yet pushed to pypi).

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    strip_prefix = "rules_closure-4a79cc6e6eea5e272fe615db7c98beb8cf8e7eb5",
+    urls = [
+        "http://mirror.tensorflow.org/github.com/bazelbuild/rules_closure/archive/4a79cc6e6eea5e272fe615db7c98beb8cf8e7eb5.tar.gz",
+        "https://github.com/bazelbuild/rules_closure/archive/4a79cc6e6eea5e272fe615db7c98beb8cf8e7eb5.tar.gz",  # 2019-09-16
+    ],
+)
+
+load("@io_bazel_rules_closure//closure:repositories.bzl", "rules_closure_dependencies")
+rules_closure_dependencies(
+    #omit_com_google_protobuf = True,
```
Comment (tolga-b) [inline WORKSPACE]: is this comment necessary?

File: wit_tensorboard/pip_package/RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# To release a new version of wit_tensorboard on PyPI:
+
+1. Ensure the updated version number have been merged to the master branch in
+wit_tensorboard/pip_package/setup.py.
+2. Clone this repository and checkout the commit that set the new version numbers.
+3. `bazel run wit_tensorboard/pip_package:build_pip_package`
```
Comment (tolga-b) [inline wit_tensorboard/pip_package/RELEASE.md]: this is separate from witwidget right?

File: witwidget/notebook/jupyter/js/lib/labplugin.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+var witwidget = require('./index');
+var base = require('@jupyter-widgets/base');
+
+module.exports = {
+  id: 'wit-widget',
+  requires: [base.IJupyterWidgetRegistry],
+  activate: function(app, widgets) {
+    widgets.registerWidget({
+      name: 'wit-widget',
+      version: '0.1.0',
```
Comment (tolga-b) [inline witwidget/notebook/jupyter/js/lib/labplugin.js]: should this be updated to actual witwidget version? it has been working like this so far.

File: tf_interactive_inference_dashboard/demo/BUILD [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    deps = [":demo"],
+)
+
+tf_web_library(
+    name = "imagedemo",
+    testonly = True,
+    srcs = [
+        "image_index.html",
+        "tf-interactive-inference-image-demo.html",
+        "@org_tensorflow_tfjs//:tf.min.js",
```
[the remark this answers is not in the document]
Comment (jameswex) [reply tf_interactive_inference_dashboard/demo/BUILD]: done

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,8 @@
+*~
```
[the remark this answers is not in the document]
Comment (jameswex) [reply .gitignore]: acknowledged

File: WORKSPACE (hunk 1 above)
Comment (jameswex) [reply WORKSPACE]: removed

File: witwidget/notebook/jupyter/js/lib/labplugin.js (hunk 3 above)
Comment (jameswex) [reply witwidget/notebook/jupyter/js/lib/labplugin.js]: going to keep it as-is and working in this PR that moves the code over. we can revisit in its own work

**Pull request merged**